crates/core/src/commands/check.rs

```diff
@@ -306,7 +306,13 @@ pub(crate) fn check_repository<P: ProgressBars, S: Open>(
 
         packs.into_par_iter().for_each(|pack| {
             let id = pack.id;
-            let data = be.read_full(FileType::Pack, &id).unwrap();
+            let data = match be.read_full(FileType::Pack, &id) {
+                Ok(data) => data,
+                Err(err) => {
+                    error!("Error reading data for pack {id} : {err}");
```

Comment:
Question: should this error log be different from the one line 318?

Reply:
Good catch, this is the next step to handle errors in check more gracefully, than currently. So I changed it a bit and added two notes about that.